docs/guides/cloud/migration-to-new-azure-images.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Migration to New Azure Images
+author: Neil Hanlon
+contributors: Steven Spencer
+tags:
+    - cloud
+    - azure
+    - microsoft azure
+    - deprecation
+---
+
+!!! info "Old Publishing Account Images Deprecated April 2024"
+    The images contained in the former publishing account (`erockyenterprisesoftwarefoundationinc1653071250513`) were marked for deprecation on April 23, 2024 with a 180-day (6 month) window until the images can no longer be used. 
+    
+    To continue using Rocky Linux on Azure, you must either migrate to the new publishing account (`resf`) or to the new Community Galleries images by following the guide below.
+
+# Migration Guide: Transitioning to New Rocky Linux Images on Azure
+
+This guide provides detailed steps on how to migrate your Azure Virtual Machines (VMs) from the deprecated Rocky Linux images to the new images under the `resf` publisher account or to utilize the Community Galleries. Following this guide will ensure a smooth transition with minimal disruption.
+
+## Before You Start
+
+- Ensure you have a current backup of your VM. While the migration process should not affect your data, having a backup is a best practice for any system changes.
+- Verify that you have the necessary permissions to create new VMs and manage existing ones in your Azure account.
+
+## Step 1: Locate Your Existing VMs
+
+Identify the VMs deployed with the old Rocky Linux images. You can do this by filtering your VMs using the old publisher account name: `erockyenterprisesoftwarefoundationinc1653071250513`.
+
+## Step 2: Prepare New VM
+
+1. **Navigate** to the Azure Marketplace.
+2. **Search** for the new Rocky Linux images under the `resf` publisher account or access the Community Galleries.
+    * Current Marketplace Links:
+        * [Rocky Linux x86_64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/resf.rockylinux-x86_64)
+        * [Rocky Linux aarch64](https://azuremarketplace.microsoft.com/en-us/marketplace/apps/resf.rockylinux-aarch64)
+    * Full instructions on accessing the Community Gallery images can be found in this [news post](https://rockylinux.org/news/rocky-on-azure-community-gallery/)
```

Comment:
this will 404 until rocky-linux/rockylinux.org#588 is merged. i.e., this is expected for now.